conda/cli/helpers.py

```diff
@@ -13,6 +13,50 @@
     _MutuallyExclusiveGroup,
 )
 
+try:
+    from argparse import BooleanOptionalAction
+except:
+    # Python < 3.9
+    from argparse import Action
+    class BooleanOptionalAction(Action):
```

Comment:
Messy to copy/paste for <= py 3.8; Implements click-style `--[no-]-repodata-zst` option pair